src/Workspaces/Core/Portable/Shared/Utilities/DocumentationComment.cs

```diff
@@ -65,6 +65,11 @@ internal sealed class DocumentationComment
         /// </summary>
         public string CompletionListCref { get; private set; }
 
+        /// <summary>
+        /// Used for TrimEachLine(string text) method, to prevent new allocation of string
+        /// </summary>
+        private static string NewLineString = Environment.NewLine;
```

Comment:
Environment.NewLine doesn't allocate a new string. If anything I think we should cache ```new string[] { NewLineString }```.

Style: private static fields should be named ```s_XXX``` and be ```readonly``` if possible.